src/screens/FeedScreen/ContentPoll.js

```diff
@@ -7,6 +7,7 @@ import PollOptions from '../../components/PollOptions';
 import PollOptionsMultipleChoice from '../../components/PollOptionsMultipleChoice';
 import useContentPoll from './hooks/useContentPoll';
 import {COLORS} from '../../utils/theme';
+import {colors} from '../../utils/colors';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been updated from `COLORS` to `colors`. Ensure that the new `colors` module provides the same color definitions as the old `COLORS` module. If not, this could lead to unexpected color changes throughout the application.
```diff
- import {COLORS} from '../../utils/theme';
+ import {colors} from '../../utils/colors';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->